Backpack/Calendar/Classes/BPKCalendarPriceLabelCell.m

```diff
@@ -80,9 +82,26 @@ - (void)prepareForReuse {
 - (void)configureWithData:(id)data {
     NSAssert([data isKindOfClass:BPKCalendarPriceLabelCellData.class], @"BPKCalendarPriceLabelCell can only be configured with data of type BPKCalendarPriceLabelCellData");
     BPKCalendarPriceLabelCellData *priceLabelCellData = (BPKCalendarPriceLabelCellData *)data;
+    self.priceLabelCellData = priceLabelCellData;
     self.priceLabel.text = priceLabelCellData.price;
     self.priceLabel.textColor = priceLabelCellData.labelStyle.textColor;
     self.priceLabel.fontStyle = priceLabelCellData.labelStyle.isBold ? BPKFontStyleTextXsEmphasized : BPKFontStyleTextXs;
+    [self updateAccessibilityLabel];
+}
+
+- (void)setAccessibilityLabel:(NSString *)accessibilityLabel {
+    self.consumerAccessibilityLabel = accessibilityLabel;
+    [self updateAccessibilityLabel];
+}
+
+-(void)updateAccessibilityLabel {
+    NSString *accessibilityLabel = self.consumerAccessibilityLabel;
+
+    if (self.priceLabelCellData.price != nil) {
+        accessibilityLabel = [NSString stringWithFormat:@"%@ %@", self.consumerAccessibilityLabel, self.priceLabelCellData.price];
+    }
+
+    [super setAccessibilityLabel:accessibilityLabel];
```

Comment:
I guess we could override the `accessibilityLabel` getter instead? Maybe that would be cleaner?

Reply:
Actually I'm pretty sure it's an improvement so I've just gone ahead and done it 👍